<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reportsDialog</class>
 <widget class="QDialog" name="reportsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Reports</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/podbicon.png</normaloff>:/podbicon.png</iconset>
  </property>
  <widget class="QGroupBox" name="configGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>951</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Configure Report</string>
   </property>
   <widget class="QPushButton" name="goPushButton">
    <property name="geometry">
     <rect>
      <x>904</x>
      <y>24</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Generate the report</string>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="endDateLabel">
       <property name="text">
        <string>End Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="startDateEdit">
       <property name="toolTip">
        <string>Select the start date of the report date range</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="startDateLabel">
       <property name="text">
        <string>Start Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="reportTypeLabel">
       <property name="text">
        <string>Report Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="reportTypeComboBox">
       <property name="toolTip">
        <string>Select the report type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="additionalDataLabel">
       <property name="text">
        <string>Additional Data:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="additionalDataComboBox">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="endDateEdit">
       <property name="toolTip">
        <string>Select the end date of the report date range</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="resultGroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>951</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Report Result</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>931</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalResultLabel">
    <property name="geometry">
     <rect>
      <x>858</x>
      <y>390</y>
      <width>81</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalLabel">
    <property name="geometry">
     <rect>
      <x>793</x>
      <y>390</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="donePushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close the reports dialog</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="clearToolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reset report configuration to defaults</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="printToolButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Print report</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/print.png</normaloff>:/print.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="exportToolButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export report to PDF file</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/exportpdf.png</normaloff>:/exportpdf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
